app/models/vm/operations.rb

```diff
@@ -5,6 +5,18 @@ module Vm::Operations
   include_concern 'Power'
   include_concern 'Lifecycle'
 
+  included do
+    supports :launch_cockpit do
+      if ipaddresses.blank?
+        unsupported_reason_add :launch_cockpit, 'Launching of Cockpit requires an IP address for the VM.'
+      end
+    end
+  end
+
+  def cockpit_url
+    "http://#{ipaddresses.first}:9090"
```

Comment:
@martinpovolny should this be built via more standard (and safe) [URI::HTTP.build](http://www.rubydoc.info/stdlib/uri/URI%2FHTTP.build)?  This can be done in a later PR.